font-tlwg.yaml

```
@@ -0,0 +1,55 @@
+package:
+  name: font-tlwg
+  version: 0.7.3
+  epoch: 0
+  description: font-tlwg
+  copyright:
+    - license: GPL
```

```suggestion
    - license: GPL-2.0-only
```

This should pass CI, confirmed it is v2 here: https://github.com/tlwg/fonts-tlwg/blob/master/GPL